src/main/java/pl/edu/zut/mad/schedule/model/inner/Schedule.java

```diff
@@ -113,6 +110,94 @@ public String getDay() {
         return day.trim();
     }
 
+    public Integer getGroupId() {
+        return groupId;
+    }
```

Comment:
I think that this getter could be replaced with `@Getter` annotation, it shouldn't affected to other getters.

Reply:
It is the only getter, that would be affected by this change. I thin using `@Getter` annotation it this case would reduce readability.